src/test/java/cloud/cholewa/game/PlayerTest.java

```diff
@@ -0,0 +1,51 @@
+package cloud.cholewa.game;
+
+import cloud.cholewa.config.ErrorDict;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class PlayerTest {
+
+    @Test
+    public void shouldThrow_IllegalArgumentException_WhenNameIsNull() {
+        //given
+
+        //when
+
+        //then
+        try {
+            new Player(null);
+            fail("Expected Illegal Argument Exception, but was not thrown");
+        } catch (IllegalArgumentException e) {
+            assertEquals(ErrorDict.PLAYER_CREATE_ERROR_CANT_BE_NULL, e.getMessage());
+        }
+    }
+
+    @Test
+    public void shouldThrow_IllegalArgumentException_WhenNameEmptyString() {
+        //given
+
+        //when
+        //then
+        try {
+            new Player("");
+            fail("Expected Illegal Argument Exception, but was not thrown");
+        } catch (IllegalArgumentException e) {
+            assertEquals(ErrorDict.PLAYER_CREATE_ERROR_CANT_BE_EMPTY_STRING, e.getMessage());
+        }
+    }
+
```

Comment:
Generalnie ok, natomiast, pomijając kwestie wrzucenia tego case'a do jednego wyjątku, możnaby użyć assertThrows z jUnit 5, np.

Throwable exceptionThatWasThrown = assertThrows(IllegalArgumentException.class, () -> {
    new Player(null)
});

assertThat(exceptionThatWasThrown.getMessage(), equalTo("Message I expected to be thrown"));

Osobiście uważam to za ładniejsze rozwiązanie, ale osobiście dlatego daje 'generalnie ok' ;)

Reply:
Skorzystałem z assertJ, chyba jest jeszcze bardziej czytelny.

Reply:
O, nie miałem okazji korzystać, ale zgodzę się w pełni - jeszcze śliczniejszy :)